libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI+Blocks.h

```diff
@@ -248,22 +246,6 @@ typedef void (^SFRestResponseBlock) (id _Nullable response, NSURLResponse * _Nul
                                               failBlock:(SFRestFailBlock)failBlock
                                           completeBlock:(SFRestArrayResponseBlock)completeBlock;
 
-/**
- * Executes a request that returns json
- * @param method the HTTP method
- * @param path the request path
- * @param queryParams the parameters of the request (could be nil)
- * @param failBlock the block to be executed when the request fails (timeout, cancel, or error)
- * @param completeBlock the block to be executed when the request successfully completes
- * @return the newly sent SFRestRequest
- */
-
-- (SFRestRequest *) performRequestWithMethod:(SFRestMethod)method
```

Comment:
Removed this API because this API doesn't allow the consumer to set body for `POST` (previously `NetworkSDK` was computing it from `queryParams`). We have documented the new way of doing it by setting body explicitly using `SFRestAPI`.